### PR TITLE
Review contributing docs for language and ease

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Checklist
- [x] All tests are passing
- [x] README / documentation was extended, if necessary
- [x] Used meaningful commit messages

### Description

This is part one of an extensive review of the contribution docs in an attempt to make it easier for new developers to get started with contributing to Solidity, 

In an attempt to make smaller PRS:

- part 2 https://github.com/ethereum/solidity/pull/5121
- part 3 https://github.com/ethereum/solidity/pull/5122

### CODE REVIEW

**Comment by lazaridiscom** [comment]
Not sure if this is the process to follow. At some points you change the writing-style. It is better to point out the weaknesses of a document, and let the owner change it (thus it keeps the writing-style of the owner). Some owners are sensitive to changes, not sure if this is the case here.

In any way, simplifying documentation relies on simplifying the underlying system first(!). The ideal documentation is `no documentation`, because the system is self-explaining.

So, instead to try to formulate something better, you simply negate(!) the need to explain. Then you delete the sentence / paragraph etc.

Applied to solidity: 

The goal would be that a capable developer would simply fetch the repo, review the code in his favorite environment (e.g. IDE), and detect immediately what is going on, with minimal explanations needed (e.g. in some readme's within the folders).

Having said this: the *.rst should possibly be changed to *.md.

File: docs/contributing.rst [hunk 1]
```diff
@@ -4,68 +4,69 @@ Contributing
...
+  <https://gitter.im/ethereum/solidity>`_ channel.
 * Fixing and responding to `Solidity's GitHub issues
   <https://github.com/ethereum/solidity/issues>`_, especially those tagged as
   `up-for-grabs <https://github.com/ethereum/solidity/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs>`_ which are
   meant as introductory issues for external contributors.
 
-Please note that this project is released with a `Contributor Code of Conduct <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_. By participating in this project - in the issues, pull requests, or Gitter channels - you agree to abide by its terms.
+This project is released with a `Contributor Code of Conduct <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_. By participating in this project - in the issues, pull requests, or Gitter channels - you agree to abide by its terms.
 
 How to Report Issues
```
Comment (lazaridiscom) [inline docs/contributing.rst]: The original version ("Please not that this...") reads better.

But!

This is anyway completely redundant. You don't need to explain to potential contributors `How to Report Issues`. Any person which does not have this common knowledge should learn within a lower grade project (something simpler than Solidity).

File: docs/contributing.rst [hunk 2]
```diff
@@ -4,68 +4,69 @@ Contributing
...
 ====================
 
-To report an issue, please use the
+To report an issue, use the
```
Comment (lazaridiscom) [inline docs/contributing.rst]: This document addresses contributors. They know where and how to file issues. If not, then this is not the right place to contribute.

Still, if you keep this, you should delete the instructions, as they are given in the issue-template.

"Use the Issue-Tracker (link) and follow the given instructions" should be enough.

File: docs/contributing.rst [hunk 3]
```diff
@@ -222,7 +223,7 @@ If the instrumentation messages did not appear, try switching the cmake flags po
     cmake .. -DCMAKE_C_COMPILER=path/to/afl-clang -DCMAKE_CXX_COMPILER=path/to/afl-clang++
     make solfuzzer
 
-Othwerise, upon execution the fuzzer will halt with an error saying binary is not instrumented:
+Otherwise, upon execution the fuzzer halts with an error saying binary is not instrumented:
```
Comment (lazaridiscom) [inline docs/contributing.rst]: expression style: Personally, I prefer `will halt`.

**Comment by lazaridiscom** [comment]
@ChrisChinchilla , is this still actual?

**Comment by ChrisChinchilla** [comment]
@lazaridiscom Thanks for the reminder, working on a rebase and actually have a lot of other work in progress around the testing sections specifically.

**Comment by ChrisChinchilla** [comment]
Although this now ended up being really small changes and maybe not worth it, seems like other PRs changed other places. Closing :)

**Pull request closed without merging**